### PR TITLE
feat(IAM Identity): adding effective account settings

### PR DESCRIPTION
**Pull request by pooraniss-ibm**
## PR summary
new endpoint to get Effective account settings, 
also has action control added to existing template calls. 

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other informatio
Screenshot 2024-08-01 at 09 40 02
n
Screenshot 2024-07-31 at 19 54 31

### CODE REVIEW

**Comment by pooraniss-ibm** [comment]
Screenshot 2024-07-31 at 19 54 31
Screenshot 2024-08-01 at 09 40 02

**Comment by pooraniss-ibm** [comment]
added screenshots of passing test cases :)

**Comment by padamstx** [comment]
@pooraniss-ibm Could you please sign off on your commit?  That is the "DCO" failure in the list of checks for the PR.  To sign off on your commit, you could use the "-s" option on `git commit`.  More details here: https://github.com/IBM/platform-services-go-sdk/blob/main/update_service.md#7-open-pr-with-your-changes

File: iamidentityv1/iam_identity_v1.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ type IamIdentityV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://iam.cloud.ibm.com"
+const DefaultServiceURL = "https://iam.test.cloud.ibm.com"
```
Comment (padamstx) [inline iamidentityv1/iam_identity_v1.go:46]: You generated the code using the staging version of the API definition again, didn't you? :joy:
You'll need to re-gen the code using the production version of the API definition

Comment (padamstx) [reply iamidentityv1/iam_identity_v1.go:46]: Also, please upgrade to the very latest version of the SDK generator before you re-gen the code using the production API defn.

**Review by padamstx — CHANGES_REQUESTED** [review]
Need to regen with the latest generator and production API defn.

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**